### ISSUE TITLE
Cron Jobs not

### ISSUE THREAD

**Issue by handypreneur** [mautic/mautic#7742]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

| Q   | A
| --- | ---
| Mautic version | 2.15.1
| PHP version | 7.0.33
| Browser | Chrome / Safari

I've tried to hold out as long as I could before posting here because I know you all are busy and some of these issues may seem redundant. I'm a researcher by heart very self-sufficient, however, I've exhausted myself trying to figure things out.

My first issues were with the "index.php" showing in the URL, I finally resolved that. My second issues we trying to get the site to use SSL, I got that resolved. Now my issue seems to be with Cron Jobs not being triggered by campaign events.

Here are the Cron Jobs I have configured in crontab:

```
* * * * * php /var/www/mautic/app/console mautic:segments:update --env=prod
* * * * * php /var/www/mautic/app/console mautic:campaigns:rebuild --env=prod
* * * * * php /var/www/mautic/app/console mautic:campaigns:trigger --env=prod
0 1 * * * php /var/www/mautic/app/console mautic:iplookup:download --env=prod
* * * * * php /var/www/mautic/app/console mautic:emails:send --env=prod
*/15 * * * * php /var/www/mautic/app/console mautic:email:fetch --env=prod
```

I'm attempting to test a Double Opt-In and when I test through submit form I get the following events for the user:

```
Contact added to campaign, TEST
Campaign membership change	
Contact was identified by Opt-In Submission Form	
Contact identified by source
Contact identified	
Contact identified
Opt-In Submission Form	Form submitted
xx.xx.xxx.xx	Accessed from IP
Contact created
```
 
... and that's were the process stops. What should be happening is the user gets added to a "pending" segment, an email is sent containing a link to confirm, once the landing page link is landed on, the contact would then get moved from the "pending" segment to a "confirmed segment.

This was working with another installation I had been trying to build with, but for some reason I cannot get this instance to perform as it should.

Any ideas? My head is literally starting to hurt.

### Log errors

There are no errors in the PHP, Apache2, or MySQL logs

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by handypreneur** [comment]
Executing **app/console mautic:campaign:trigger** manually triggers and processese the batch of new contacts. So obviously the campaigns are working but whatever is supposed to trigger them, cron jobs, does not seem to be doing so.

**Comment by handypreneur** [comment]
Disregard. I now have it working. Sometimes you just have to step away and get you mind free of the clutter. Once the clutter is cleared, you can think from another perspective -- resolved.

**Comment by Prosp3r** [comment]
Hi, how did you get it resolved ?
I've been trying to get mine installed via softacuous to send campaign emails without success.
Sample emails get delivered without a hitch so my email provider SES works fine.
But campaign emails stop just like you described.
I've deleted Cache, removed segments processes like described in other issues. Nothing seems to work.
I wish there was a fire proof step by step to get Mautic working and campaigns like double opting working hitch free.

**Comment by handypreneur** [comment]
Prosp3r,

It had to do with my Cron Jobs. I'm not sure what you're using for Crons or how your environment is setup, but mine is pretty simple and I've gotten it so it's fully operational.

Here's what I have for crons, feel free to copy them if you'd like and paste them into "crontab -e"

```
* * * * * php /var/www/mautic/app/console mautic:segments:update
* * * * * php /var/www/mautic/app/console mautic:campaigns:rebuild
* * * * * php /var/www/mautic/app/console mautic:campaigns:trigger
0 1 * * * php /var/www/mautic/app/console mautic:iplookup:download
* * * * * php /var/www/mautic/app/console mautic:emails:send
```

These Cron Jobs will kick off once per minute so give it at least that amount of time for things to work as expected.

**Comment by handypreneur** [comment]
Also, I followed many articles on the installation of Mautic and many are missing information and the proper steps to get it installed and secured properly; in other words, that are incomplete. I spent 3 days (literally) getting my installation to the point where it should be by taking tidbits of information from around the web and piecing it altogether.

Now I have a Lightsail snapshot that is setup right to the point on configuring Mautic at the web interface, and one where Mautic is completely configured for my needs. I did this mostly for testing so I didn't have to keep going through all of the necessary steps to get it working and secured.
